recipes/luau/all/conanfile.py

```diff
@@ -25,72 +30,80 @@ class LuauConan(ConanFile):
         "with_cli": False,
         "with_web": False,
     }
-    generators = "cmake"
 
     @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def _minimum_cpp_standard(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16",
```

Comment:
Can this be removed since the `check_min_vs` call exists below?

Reply:
I agree with you.
It's my fault.

Reply:
fixed